src/index.js

```diff
@@ -0,0 +1,55 @@
+import './style.css';
+import showTasks from './showTasks.js';
+import { addToLocalStorage } from './setLocalStorage.js';
+import getTasksFromLocalStorage from './getTasks.js';
+import deleteTodo from './deleteOne.js';
+import removeCompletedTasks from './clearCompleted.js';
+
+const input = document.querySelector('.text');
+const form = document.getElementById('form');
+const removeCompleted = document.querySelector('.clear-completed');
+
+function clearInput() {
+  input.value = '';
+}
+
+const addTodoTask = (e) => {
+  const tasks = getTasksFromLocalStorage();

```

Comment:
Good job on declaring local variables as global variables pose the risk of overwriting other scripts.
The whole codebase looks DRY.